baseplate/experiments/variant_sets/range_variant_set.py

```diff
@@ -0,0 +1,68 @@
+from .base import VariantSet
+
+
+class RangeVariantSet(VariantSet):
+    """ Variant Set designed to take fixed bucket ranges.
+    This VariantSet allows manually setting bucketing ranges.
+    It takes in a variant name, then the range of buckets in
+    that should be assigned to that variant. This enables user-defined
+    bucketing algorithms, as well as simplifies the ability to adjust
+    range sizes in special circumstances.
+    """
+
+    def __init__(self, variants, num_buckets=1000):
+        """ :param list variants: Array of dicts, each containing the keys 'name',
```

Comment:
i don't think you can docstring __init__ here can you? does this show up in the docs when building locally? those params would probably go in the class docstring.